<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AuditClientWidget</class>
 <widget class="QWidget" name="AuditClientWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>809</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AuditClientWidget</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>681</width>
     <height>421</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="centralWidget">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Connexioin au serveur</string>
      </property>
      <widget class="QWidget" name="horizontalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>7</x>
         <y>26</y>
         <width>661</width>
         <height>171</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <item row="2" column="2">
             <widget class="QLineEdit" name="lineEditAdresseIP">
              <property name="text">
               <string>172.0.0.1</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Adresse IP du serveur :</string>
              </property>
             </widget>
            </item>
            <item row="3" column="2">
             <widget class="QLineEdit" name="lineEditPort">
              <property name="text">
               <string>1025</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Num√©ro de port du serveur :</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="3" column="1">
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QPushButton" name="pushButtonQuitter">
              <property name="maximumSize">
               <size>
                <width>80</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Quitter</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>2</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonConnexion">
              <property name="text">
               <string>Connexion</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Etat de la connexion</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="textEditEtat"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBoxInformationsClient">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="title">
       <string>Informations Client</string>
      </property>
      <widget class="QWidget" name="horizontalLayoutWidget_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>661</width>
         <height>171</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QPushButton" name="pushButtonOrdinateur">
            <property name="text">
             <string>Nom de l'ordianteur distant</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonUtilisateur">
            <property name="text">
             <string>Nom de l'utilisateur</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonArchitecture">
            <property name="text">
             <string>Architecture processeur</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonOs">
            <property name="text">
             <string>Os de l'ordinateur</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QLineEdit" name="lineEditArchitecture">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditUtilisateur"/>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditOs"/>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditOrdinateur"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonQuitter</sender>
   <signal>clicked()</signal>
   <receiver>AuditClientWidget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>140</x>
     <y>210</y>
    </hint>
    <hint type="destinationlabel">
     <x>40</x>
     <y>242</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
